specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2021-11-20-preview/examples/ElasticSans_Create_MaximumSet_Gen.json

```diff
@@ -0,0 +1,98 @@
+{
+  "parameters": {
+    "subscriptionId": "aaaaaaaaaaaaaaa",
+    "resourceGroupName": "rgelasticsan",
+    "elasticSanName": "0Dk_bd2172_UFvY6Y",
+    "parameters": {
+      "properties": {
+        "sku": {
+          "name": "Standard_LRS",
+          "tier": "Standard"
+        },
+        "availabilityZones": [
+          "aaaaaaaaa"
+        ],
+        "baseSizeTB": 12,
+        "extendedSizeTB": 6
+      },
+      "location": "aaaaa",
+      "tags": {
+        "key3755": "aaaaaaaaa"
+      }
+    },
+    "api-version": "2021-11-20-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "sku": {
+            "name": "Standard_LRS",
+            "tier": "Standard"
+          },
+          "availabilityZones": [
+            "aaaaaaaaa"
+          ],
+          "provisioningState": "Invalid",
+          "baseSizeTB": 12,
+          "extendedSizeTB": 6,
+          "totalSizeTB": 7,
+          "provisionedIops": 26,
+          "provisionedMBps": 10
+        },
+        "systemData": {
+          "createdBy": "aaaaaaaaaaaaaaaa",
+          "createdByType": "User",
+          "createdAt": "2022-03-09T06:06:58.232Z",
+          "lastModifiedBy": "aaaaaaaaaaaaaa",
+          "lastModifiedByType": "User",
+          "lastModifiedAt": "2022-03-09T06:06:58.232Z"
+        },
+        "location": "aaaaa",
+        "id": "aaaaaaaaaaa",
```

Comment:
>"id" [](http://example.com/codeflow?start=8&length=4)

Please use a fully qualified id, here's an example: [https://github.com/Azure/azure-rest-api-specs/blob/ca0335b44b4eca2c5b5673ee2c58a87e524b669f/specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2021-10-01/examples/createKey.json#L17](https://github.com/Azure/azure-rest-api-specs/blob/ca0335b44b4eca2c5b5673ee2c58a87e524b669f/specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2021-10-01/examples/createKey.json#L17)

Reply:
Make sure to update this on all your examples.

Reply:
Updated.